lib/mumukit/metatest/checker.rb

```diff
@@ -64,7 +64,7 @@ def build_success_output(builder, _example, input)
 
     # Implementors should override this method if they want access to
     # the error details and produce more complex test results
-    def build_error_output(builder, input, _example, error)
```

Comment:
:ok_hand: This change looks consistent with https://github.com/mumuki/mumuki-gobstones-runner/blob/c6e8fb271769136bbf63984e54957d8455b84dc1/lib/render/with_renderer.rb#L11

It looks like I simply swap parameters by mistake